nano/templates/air_alarm.tmpl

```diff
@@ -0,0 +1,199 @@
+#define AALARM_MODE_REPLACEMENT	4 //sucks off all air, then refill and swithes to scrubbing
+#define AALARM_MODE_OFF			5
+#define AALARM_MODE_FLOOD		6 //Emagged mode; turns off scrubbers and pressure checks on vents
+
+#define AALARM_SCREEN_MAIN		1
+#define AALARM_SCREEN_VENT		2
+#define AALARM_SCREEN_SCRUB		3
+#define AALARM_SCREEN_MODE		4
+#define AALARM_SCREEN_SENSORS	5
+
+ -->
+ <div class='notice'>
+	{{if siliconUser}}
+		<div class="itemContentSmall">
+			Interface Lock:
+		</div>
+		<div class="itemContentFull">
+			{{:~link('Engaged', 'locked', {'toggleaccess' : 1}, locked ? 'selected' : null)}}{{:~link('Disengaged', 'unlocked', {'toggleaccess' : 1}, (locked ? null : 'selected'))}}
+		</div>
+	{{else}}
+		{{if locked}}
+			Swipe an ID card to unlock this interface.
+		{{else}}
+			Swipe an ID card to lock this interface.
+		{{/if}}
+	{{/if}}
+	<div class="clearBoth"></div>
+</div>
+<H1>Air Status</H1>
+{{if environment_data}}
+	{{for environment_data}}
+		{{:name}}
+		{{if danger_level == 2}}
+			<span class='bad'>
+		{{else danger_level == 1}}
+			<span class='average'>
+		{{else}}
+			<span class='good'>
+		{{/if}}
+		{{:~round(value*100)/100}}
+		</span>{{:unit}}<br>
+	{{/for}}
+	Local Status: {{if danger_level == 2}}
+			<span class='bad'>DANGER: Internals Required
+		{{else danger_level == 1}}
+			<span class='average'>Caution
+		{{else}}
+			<span class='good'>Optimal
+		{{/if}}
+		</span><br>
+    Area Status: {{if atmos_alarm}}<span class='bad'>Atmosphere alert in area</span>{{else fire_alarm}}<span class='bad'>Fire alarm in area</span>{{else}}No alerts{{/if}}
+{{else}}
+	<span class='bad'>Warning: Cannot obtain air sample for analysis.</span>
+{{/if}}
+{{if dangerous}}
+	<br><span class='bad'>NOTICE: Safety measures nonfunctional. Device may exhibit abnormal behavior.</span>
+{{/if}}
+<HR>
+{{if (!locked || siliconUser)}}
+    {{if screen != 1}}
+        <div class="item">{{:~link('Main Menu', null, { 'screen' : 1})}}</div>
+    {{/if}}
+    {{if screen == 1}}
+        <div class="item">
+            {{if atmos_alarm}}
+                {{:~link('Reset - Area Atmospheric Alarm', null, { 'atmos_reset' : 1})}}
+            {{else}}
+                {{:~link('Activate - Area Atmospheric Alarm', null, { 'atmos_alarm' : 1})}}
+            {{/if}}
+        </div>
+        <hr>
+            <div class="item">
+                {{:~link('Scrubbers Control', null, { 'screen' : 3})}}
+            </div>
+            <div class="item">
+                {{:~link('Vents Control', null, { 'screen' : 2})}}
+            </div>
+            <div class="item">
+                {{:~link('Set Environmental Mode', null, { 'screen' : 4})}}
+            </div>
+            <div class="item">
+                {{:~link('Sensor Settings', null, { 'screen' : 5})}}
+            </div>
+        <hr>
+            {{if mode==3}}
+                {{:~link('PANIC SIPHON ACTIVE - Turn siphoning off', null, { 'mode' : 1}, null, 'redButton')}}
+			{{else}}
```

Comment:
Probably. Thankfully it's HTML, and indentation means nothing to the actual code, will put a fix in with whatever inevitable bug fix crops up sooner or later.